### ISSUE TITLE
Small Changes / requests

### ISSUE THREAD

**Issue by MonkeyWrench123** [J05HI/DeezLoadr#5]
Hi Jo5hi, wow you work quick.

If you get the time is it possible to implement the following?

- Check if file already exists in location and skip (would be great if this was super quick)
- Option to replace some invalid characters in folder/track name with _ like itunes instead of just removing them ('?\/")
- Mechanism to load multiple links (would be great if supported long lists)

Great work

**Comment by J05HI** [comment]
@MonkeyWrench123 

Thank you very much!!!


The skipping will be implemented as soon as possible :)

Can you please give me some examples of iTunes track names with these _.
I think this looks really strange in the filename.

I don't know how to implement a meachanism to load multiple links in a console app :(
Maby we should wait for the GUI here.

**Comment by MonkeyWrench123** [comment]
Beat you to the new link by the way  ;)

I've found some old code where it was implemented along side removing any Diacritics, not really a biggy, would just be nice to tie up with an existing library so it doesn't go off and get loads of duplicates.

function fixName (input, file) {
  const regEx = file ? /[,\/\\:*?"<>|]/g : /[\/\\"<>*?:|]|\.$/g;
  return removeDiacritics(input.replace(regEx, '_'));
}

So 01 What do you mean? would just become What do you mean_

Could you look for a file in the current directory, if found, loop through each line?, could also save to the file with current link (to rescan in future). This is all very easy of me to say, but I have no idea how would it be done :P.

**Comment by J05HI** [comment]
@MonkeyWrench123
Okay I'll implement it but let the user choose what he likes better :)

**Comment by MonkeyWrench123** [comment]
Sorry for taking ages, I just checked. Itunes seems to replace every unusable symbol with an _

**Comment by J05HI** [comment]
@MonkeyWrench123
Okay really strange

**Comment by MonkeyWrench123** [comment]
Yeah it is a bit strange, I think it kinda makes sense though, I guess would be less chance of getting duplicate artists in the same folder (if a name is similar but with special characters)

**Comment by J05HI** [comment]
<img width="1009" alt="1" src="https://user-images.githubusercontent.com/16399787/33243639-6b2894cc-d2ea-11e7-9efe-5a29688c6a37.PNG">

I've currently purchased this random song on iTunes 😄 
See what it does, you were right :)

<img width="602" alt="2" src="https://user-images.githubusercontent.com/16399787/33243640-6ebbf5d4-d2ea-11e7-88ec-1d15b6decc2a.PNG">

**Comment by J05HI** [comment]
@MonkeyWrench123 
> Check if file already exists in location and skip (would be great if this was super quick)

Added in [https://github.com/J05HI/DeezLoadr/commit/0f91a69f6ee64e00cdacc9cf576dece38eeda82c](https://github.com/J05HI/DeezLoadr/commit/0f91a69f6ee64e00cdacc9cf576dece38eeda82c) :)

**Comment by J05HI** [comment]
Other will be implemented in v1. 3.0

**Comment by troufiniou** [comment]
For the multiple link we can just put multiple id

**Comment by J05HI** [comment]
@troufiniou 
How do you mean that?

**Comment by troufiniou** [comment]
Like that 

Imagine I want that
http://www.deezer.com/fr/album/8857671
http://www.deezer.com/fr/album/13954198

At the launch of the app

Put Deezer id : 8857671,13954198

And voila !

**Comment by J05HI** [comment]
@troufiniou 
Yeah comma seperated could be a solution :)

**Comment by MonkeyWrench123** [comment]
I understand batch downloading is very low priority :), but after a bit of a look around, perhaps forerunnerdb could be used?  It allows you to create a database, run Queries, save and load to a file.

**Comment by J05HI** [comment]
@MonkeyWrench123 
I don't need a database.
Will integrate the comma seperated urls in v1.3.0

**Comment by MonkeyWrench123** [comment]
Yeah that would be brill. Have you seen headphones? https://github.com/rembo10/headphones integrating into something like this could be cool

**Comment by RandomNinjaAtk** [comment]
@MonkeyWrench123 Lidarr would be better! https://github.com/lidarr/Lidarr

**Comment by J05HI** [comment]
They won't add it if I do a pull request...

**Comment by J05HI** [comment]
I'll improve this app as much as I can :)

**Comment by J05HI** [comment]
# Project moved!
## New project: [https://github.com/J05HI/SMLoadr](https://github.com/J05HI/SMLoadr)
## Please delete all of your forks!!!
## I'll delete this repo soon!
